### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by vsumner**
Update the remaining dependencies not covered by other PRs.

`chalk` from `2.4.1` to `4.0.0`
`@shopify/useful-types` from `1.1.2` to `2.1.4`
`@types/webpack` from `4.4.27` to `4.41.11`
`change-case` from `3.0.1` to `4.1.1`
`chokidar` from `2.0.3` to `3.3.1`

Adds `upper-case-first@2.0.1` which was split out from `change-case`.

### CODE REVIEW

**Review by alexandcote — COMMENTED** [review]
I think the `chalk` import need to be renamed to: 
```ts
import * as chalk from 'chalk'
```

More info under the Minor Breaking change [here](https://github.com/chalk/chalk/releases/tag/v3.0.0).

**Comment by vsumner** [comment]
> I think the `chalk` import need to be renamed to:
> 
> ```ts
> import * as chalk from 'chalk'
> ```
> 
> More info under the Minor Breaking change [here](https://github.com/chalk/chalk/releases/tag/v3.0.0).

Thanks! I missed that one.

File: packages/graphql-mini-transforms/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and from `v0.14.0`, this project adheres to [Semantic Versioning](http://semver.
 
 ### Changed
 
+- Update dependencies [[#110](https://github.com/Shopify/graphql-tools-web/pull/110)]
```
[the remark this answers is not in the document]
Comment (vsumner) [reply packages/graphql-mini-transforms/CHANGELOG.md]: Updated.

File: packages/graphql-typescript-definitions/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ and from `v0.14.0`, this project adheres to [Semantic Versioning](http://semver.
 
 ### Changed
 
+- Update dependencies (`chalk`, `@shopify/useful-types`, `@types/webpack`, `change-case`, `chokidar`, `upper-case-first`) [[#110](https://github.com/Shopify/graphql-tools-web/pull/110)]
```
Comment (alexandcote) [inline packages/graphql-typescript-definitions/CHANGELOG.md:10]: You didn't bump `@shopify/useful-types` and `@types/webpack` in this package

**Review by alexandcote — APPROVED** [review]

File: packages/graphql-mini-transforms/CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ and from `v0.14.0`, this project adheres to [Semantic Versioning](http://semver.
 
 ### Changed
 
+- Update dependencies (`chalk`, `@shopify/useful-types`, `@types/webpack`, `change-case`, `chokidar`, `upper-case-first`) [[#110](https://github.com/Shopify/graphql-tools-web/pull/110)]
```
[the remark this answers is not in the document]
Comment (vsumner) [reply packages/graphql-mini-transforms/CHANGELOG.md:10]: copy pasta error. thanks

**Pull request merged**